addons/hide-signatures/addon.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Hide signatures in forums",
+  "description": "Hides people's signatures in the forums.",
+  "credits": [
+    {
+      "name": "Verixion",
+      "link": "https://scratch.mit.edu/users/Verixion/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["topics"],
+      "runAtComplete": true
+    }
+  ],
+  "settings": [
+    {
+      "name": "Hide signatures by default",
```

Comment:
When would you not want to hide them by default if you have this addon?

Reply:
Idk